DefaultSettings.php

```diff
@@ -53,7 +53,7 @@
 	'media',
 	'datatables',
 	'spreadsheet',
-
+	'gantt'
```

Comment:
A blank line between this and the following note as before will be fine

Reply:
Because of consitency removed the blank line
See: 56

Commit: 3aeae38c7300a22b8e6697a7fac90ef400a5a0b4